### ISSUE TITLE
Incorrect che version installed using chectl

### ISSUE THREAD

**Issue by rajivnathan** [eclipse-che/che#18261]
### Describe the bug

Downloaded the chectl 7.20.1 release from https://github.com/che-incubator/chectl/releases/tag/20201028124946 and tried installing on Openshift 4.5.2 but it installs 7.19.2 instead. (As indicated by both the chectl output and Openshift console)

Note: I also tried with the `-i quay.io/eclipse/che-operator:7.20.1` option but it still installs 7.19.2

chectl output:
```
./chectl server:deploy -n toolchain-che -p openshift                                                                                                                              2 ✘  10:50:32
› Current Kubernetes context: 'admin'
› Installer type is set to: 'olm'
  ✔ Verify Kubernetes API...OK (it's OpenShift)
  ✔ 👀  Looking for an already existing Eclipse Che instance
    ✔ Verify if Eclipse Che is deployed into namespace "toolchain-che"...it is not
  ✔ ✈️  Openshift preflight checklist
    ✔ Verify if oc is installed...done.
    ✔ Verify if openshift is running...done.
    ✔ Check OpenShift version: 4.x
    ✔ Check Kubernetes version: Unknown.
  ✔ Verify Openshift oauth....done.
  ✔ Start following logs
    ✔ Start following Operator logs...done
    ✔ Start following Eclipse Che logs...done
    ✔ Start following Postgres logs...done
    ✔ Start following Keycloak logs...done
    ✔ Start following Plugin registry logs...done
    ✔ Start following Devfile registry logs...done
  ✔ Start following events
    ✔ Start following namespace events...done
  ✔ 🏃‍  Running Olm installaion Eclipse Che
    ✔ Check if OLM is pre-installed on the platform...done.
    ✔ Copying operator resources...done.
    ✔ Create Namespace (toolchain-che)...done.
    ✔ Create operator group...created new one.
    ✔ Configure context information...done.
    ✔ Create operator subscription...created new one.
    ✔ Wait while subscription is ready...done.
    ✔ Approve installation...done.
    ✔ Wait while operator installed...done.
    ✔ Create the Custom Resource of type checlusters.org.eclipse.che in the namespace toolchain-che...done.
  ✔ ✅  Post installation checklist
    ✔ PostgreSQL pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✔ starting...done.
    ✔ Keycloak pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✔ starting...done.
    ✔ Devfile registry pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✔ starting...done.
    ✔ Plugin registry pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✔ starting...done.
    ✔ Eclipse Che pod bootstrap
      ✔ scheduling...done.
      ✔ downloading images...done.
      ✔ starting...done.
    ✔ Eclipse Che status check
  ✔ Retrieving Keycloak admin credentials...done
  ✔ Retrieving Che self-signed CA certificate... done
  ✔ Prepare post installation output...done
  ✔ Show important messages
    ✔ ❗ [WARNING] OpenShift OAuth is turned off, because there is no any identity providers configured. https://docs.openshift.com/container-platform/latest/authentication/understanding-identity-provider.html#identity-provider-overview_understanding-
identity-provider
    ✔ [ACTION REQUIRED] Please add Che self-signed CA certificate into your browser: /Users/rajiv/cheCA.crt.
   Documentation how to add a CA certificate into a browser: https://www.eclipse.org/che/docs/che-7/overview/importing-certificates-to-browsers/
    ✔ Eclipse Che 7.19.2 has successfully installed.
    ✔ Users can access the Dashboard at   : https://che-toolchain-che.apps.<obfuscated>
    ✔ Plugins can be viewed at            : https://plugin-registry-toolchain-che.apps.<obfuscated>/v3
    ✔ Devfiles can be viewed at           : https://devfile-registry-toolchain-che.apps.<obfuscated>
    ✔ Identity provider can be accessed at: https://keycloak-toolchain-che.apps.<obfuscated>/auth
    ✔ Product documentation is at         : https://www.eclipse.org/che/docs/
    ✔ Identity Provider credentials       : <obfuscated>.
Command server:deploy has completed successfully in 06:26.
```

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

```
./chectl version                                                                                                                                                                    ✔  10:48:56
chectl/7.20.1 darwin-x64 node-v10.23.0
```

### Steps to reproduce

1. Download chectl release from https://github.com/che-incubator/chectl/releases/tag/20201028124946
2. Run the following command `./chectl server:deploy -n toolchain-che -p openshift`

### Expected behavior

Expect to see     
✔ Eclipse Che 7.20.1 has successfully installed.

Actual output shows     
✔ Eclipse Che 7.19.2 has successfully installed.

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [x] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

`oc version` output:
```
Client Version: openshift-clients-4.5.0-202006231303.p0-4-gb66f2d3a6
Server Version: 4.5.2
Kubernetes Version: v1.18.3+b74c5ed
```

### Installation method
   - [x] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [x] macOS
   - [ ] Cloud
       - [x] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by ibuziuk** [comment]
@tolusha @mkuznyetsov any ideas? Could you please take a look?

**Comment by tolusha** [comment]
@rajivnathan 
Accordingly to the logs the `olm` installer is used. 
`› Installer type is set to: 'olm'`
Unfortunately he latest available version in the catalog is 7.19.2 cc @mkuznyetsov 

To install 7.20.1 pls use `--installer operator` flag.

**Comment by ibuziuk** [comment]
> Unfortunately he latest available version in the catalog is 7.19.2 cc @mkuznyetsov

@tolusha but this is unexpected I believe, so not sure why the issue is closed

**Comment by rajivnathan** [comment]
Thanks @tolusha that worked. 🙌 

I think the README could be improved to explain the differences between the installer types.
